ssh/logger.go

```diff
@@ -12,7 +13,12 @@ var (
 )
 
 func init() {
-	logFile, err := os.OpenFile(`/tmp/onetouch.log`, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0600)
+	user, err := user.Current()
```

Comment:
this fails in some systems, especially before go 1.11. so we need to fallback. see utils/helpers.go for an example or we can store this log somewhere in the user's home

Reply:
ok so i suppose best would be store users logs in $HOME, best in ~/.onetouch.log ?

Reply:
I think `~/.onetouch/logs/` would be a better place